activator.properties

```diff
@@ -0,0 +1,8 @@
+name=hello-akka-scala
```

Comment:
Not sure if this is needed? Just followed this guide (and `step 6` says to add a activator.properties file to the root of the project): http://developer.lightbend.com/blog/2017-03-17-how-to-migrate-activator-template/index.html

@eed3si9n  Is `activator.properties` used somehow or can it be removed?